src/components/MeetTheTeam/index.jsx

```diff
@@ -27,23 +28,60 @@ const MeetTheTeam = () => {
           slidesToScroll={settings.slidesToScroll}
         >
           <div>
-            div 1
             <Container>
             <Row>
               <Col className="avatar" sm={4}></Col>
               <Col sm={8}>
-                <div>NAME SURNAME</div>
+                <div>NAME SURNAME1</div>
                 <div>GitHub</div>
               </Col>
             </Row>
             </Container>
           </div>
           <div>
-            div 2
+          <Container>
```

Comment:
Can you use the div instead of container in these places? Container should be the outer div not the inside other divs.